### PR TITLE
Update README to reflect FOLIO

### PR DESCRIPTION
**Pull request by marlo-longley**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -141,19 +141,20 @@ cat record.json | bundle exec traject -c lib/traject/config/folio_config.rb -s r
...
-## environments
-### Symphony ILS (Sirsi)
-MARC binary data is dumped into files (each containing ~500k records) on the Symphony servers. These dumps happen hourly (containing every changed MARC record during that calendar day), nightly (containing every record changed the previous day), and monthly (containing every exportable MARC record in Symphony). Hourly and nightly dumps also include a `del` file, containing a catkey-per-line of records that have been deleted or retracted.
+It's also possible to index a group of druids, mimicking the process from SDR.
+`SOLR_URL=http://localhost:8983/solr/blacklight-core bundle exec traject -i xml -c lib/traject/config/sdr_config.rb druidslist.txt`
+You can create a `druids.txt` file containing a list of newline delimitd druids.
 
-Additional data comes from a course reserves data dump, also on the Symphony servers. Course reserves files are pipe `|` separated values (PSV) files which are read in during the indexing process and used to enhance MARC records during the transform process.
+### Data sources 
+Our catalog data comes from queries to the FOLIO Postgres database.
```
Comment (jcoyne) [inline README.md:149]: And from SDR. Maybe this statement should be under the FOLIO heading?

Comment (marlo-longley) [reply README.md]: Thanks @jcoyne. Updated.

File: README.md [hunk 2]
```diff
@@ -141,21 +141,22 @@ cat record.json | bundle exec traject -c lib/traject/config/folio_config.rb -s r
...
-## environments
-### Symphony ILS (Sirsi)
-MARC binary data is dumped into files (each containing ~500k records) on the Symphony servers. These dumps happen hourly (containing every changed MARC record during that calendar day), nightly (containing every record changed the previous day), and monthly (containing every exportable MARC record in Symphony). Hourly and nightly dumps also include a `del` file, containing a catkey-per-line of records that have been deleted or retracted.
+It's also possible to index a group of druids, mimicking the process from SDR.
+`SOLR_URL=http://localhost:8983/solr/blacklight-core bundle exec traject -i xml -c lib/traject/config/sdr_config.rb druidslist.txt`
+You can create a `druids.txt` file containing a list of newline delimitd druids.
 
-Additional data comes from a course reserves data dump, also on the Symphony servers. Course reserves files are pipe `|` separated values (PSV) files which are read in during the indexing process and used to enhance MARC records during the transform process.
+### Data sources 
+Course reserve information is retrieved from FOLIO and associated with items and holdings for retrieval in the indexing process.
```
Comment (jcoyne) [inline README.md:149]: Shouldn't this be under FOLIO too? I don't think SDR items can have course reserves.

File: README.md [hunk 3]
```diff
@@ -141,21 +141,22 @@ cat record.json | bundle exec traject -c lib/traject/config/folio_config.rb -s r
...
-
-### SDR
-The indexing machines also have scheduled cron tasks for loading data from purl-fetcher into a kafka topic. This task records a state file (in `./tmp`) that contains the timestamp of the most recent entry from purl-fetcher that was processed. Every minute, the cron task runs, retrieves the purl-fetcher changes since that most recent timestamp, and adds the message to a kafka topic.
+This traject config uses the special setting `SKIP_EMPTY_ITEM_DISPLAY`, which tells the indexer to skip or not skip empty item_display fields. anything greater than -1 will skip. tests are set to use `-1` unless otherwise configured.
 
+## environments
 ### FOLIO
-Data is read directly from the postgres database underlying FOLIO, using a custom SQL query stored in the `FolioPostgresReader`. To mimic this activity in local development, one can SSH tunnel to the FOLIO database and use the `#find_by_catkey` helper method:
+Our catalog data comes from queries to the FOLIO postgres database and from SDR.
```
Comment (jcoyne) [inline README.md:157]: If it's from SDR, then it shouldn't be under the FOLIO heading.

**Review by jcoyne — CHANGES_REQUESTED** [review]

**Review by jcoyne — APPROVED** [review]

**Pull request merged**